encryption/migrations/migrations.go

```diff
@@ -1,29 +1,29 @@
-// Code generated by go-bindata. DO NOT EDIT.
+// Code generated by go-bindata.
```

Comment:
We need to use the same tools to generate it. It looks like it's not the case now.

Please check `make install-dev`. It might need updating because `protoc-gen-go@1.3.1` is used while I saw we started using 1.3.2 in some other lib.